namespaces/Representor.md

```diff
@@ -0,0 +1,44 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Resource (Element)
+
+The Resource represents a Hypermedia Resource, with its available transitions and attributes.
+
+### Properties
+
+- `element`: resource (string, fixed)
+- `attributes` (object)
+    - `transitions` - (array[Transition]) - The available transitions from this resource.
+    - `resources` - (array[Resource]) - Any embedded (resources) from this resource.
+    - `relation` - (string) - An optional relation to this resource, applicable when embedded in another resource.
+- `content` (object) - Any properties of the resource.
+
+## Transition (Object Element)
+
+A transition is an available progression from one state to another state.
+
+### Properties
+
+- `element`: transition (string, fixed)
+- `attributes` (object, required)
+    - `relation` - (string, required) - The relation of this transition from the current resource.
+    - `parameters` ([Object Type][]) - A collection of parameters for this transition.
+    - `attributes` ([Object Type][]) - A collection of possible attributes for this transition.
```

Comment:
Thinking about this even more – if you do not want elements in plain attributes hash – _maybe_ the href should be attribute and content of the transition SHOULD be input properties – parameters and attributes elements.

But hey @smizell look at MSON namespace – I am having elements in attributes object too ! – https://github.com/refractproject/refract-spec/blob/master/namespaces/mson-namespace.md#properties (actually there is a typo there and *T* should be *MSON Element* :stuck_out_tongue: )